app/assets/js/rtMedia.js

```diff
@@ -11,6 +11,10 @@ function apply_rtMagnificPopup( selector ) {
 		}
 		if ( typeof( rtmedia_lightbox_enabled ) != 'undefined' && rtmedia_lightbox_enabled == '1' ) { // If lightbox is enabled.
 
+			var old_gallery_media;
+			var current_page;
```

Comment:
Please check if we need to initialize variable because only declared variable will have "undefined" value.

Reply:
I have checked, There is no need to intialize when declaring.